### ISSUE TITLE
[Bug]: 

### ISSUE THREAD

**Issue by hivjSiricnak** [Resourceful-Bees/ResourcefulBees#337]
### Bug Description

When a bee sting (probably) any mob from the "Ice and Fire" it crash the game. I met with this when I went into a desert then the game crahed when a death worm got stinged.

### How to Reproduce?

1. hawe any resource bee at least 2 if the first is get instantly killed
2. summon any agressive "Ice and Fire" mob (death worm, myrmex, dreat thrall...; I stopped looking after the thrall)
3. let the bee got hit 
4. when the bee hits back the game crash

### Version

0.10.6

### Mod Loader Version

1.16.5 - 36.2.34

### Acknowledgement

- [X] I agree that I read the terms at the top of the page.

**Comment by Vaerys-Dawn** [comment]
crash log please

**Comment by hivjSiricnak** [comment]
[crash-2022-04-22_05.16.08-client.txt](https://github.com/Resourceful-Bees/ResourcefulBees/files/8551351/crash-2022-04-22_05.16.08-client.txt)

**Comment by Epic428** [comment]
nothing about that crash log indicates an issue with Resourceful Bees. the error lies within the renderer for the ice and fire deathworm. This would be an issue with their mod. also iirc bees only sting players.

**Comment by hivjSiricnak** [comment]
at this moment I tried what if I only use just this two mod and the game did'nt crashed when a "Ice and fire" mob got stinged by the bees

**Comment by Epic428** [comment]
```
Description: Rendering entity in world

java.lang.IllegalArgumentException: bound must be positive
```
```
-- Head --
Thread: Render thread
Stacktrace:
	at java.util.Random.nextInt(Random.java:388) ~[?:1.8.0_51] {}
	at net.minecraft.client.renderer.entity.model.EntityModel.getRandomModelPart(SourceFile:536) ~[?:?] {re:mixin,re:classloading,pl:mixin:APP:piercearrow.mixins.json:client.EntityModelMixin,pl:mixin:A}
	at baguchan.piercearrow.client.layer.LivingStuckInBodyLayer.render(LivingStuckInBodyLayer.java:36) ~[?:1.16.5-1.0.0] {re:classloading,pl:runtimedistcleaner:A}
	at baguchan.piercearrow.client.layer.LivingStuckInBodyLayer.func_225628_a_(LivingStuckInBodyLayer.java:19) ~[?:1.16.5-1.0.0] {re:classloading,pl:runtimedistcleaner:A}
	at net.minecraft.client.renderer.entity.LivingRenderer.func_225623_a_(LivingRenderer.java:125) ~[?:?] {re:classloading,pl:accesstransformer:B,pl:runtimedistcleaner:A,re:mixin,pl:accesstransformer:B,pl:runtimedistcleaner:A}
	at net.minecraft.client.renderer.entity.MobRenderer.func_225623_a_(SourceFile:43) ~[?:?] {re:classloading}
	at net.minecraft.client.renderer.entity.MobRenderer.func_225623_a_(SourceFile:18) ~[?:?] {re:classloading}
-- Entity being rendered --
Details:
	Entity Type: iceandfire:deathworm (com.github.alexthe666.iceandfire.entity.EntityDeathWorm)
```

it's literally an ice and fire issue. the only reference to our mod in the entire crash log is in the list of mods loaded. creating a test case with both of our mods won't change anything other than the fact the bug exists on ice and fire's side not our side.

**Comment by hivjSiricnak** [comment]
I think I found out the problem and sorry for the trouble. I fokused on the bee stinging back, but it seams this must be an issue by a mod wich is makes the arros shown on others both players and mobs. And when the be sting the mob it want to place an arrow or sting on it but this is crashis the game. Thanks for the help and now I'm gointo mention this problem to both side. I'm happy I dont nead to take off this mod by this problem. But first I hawe to test it.
